docs/source/install.rst

```diff
@@ -16,44 +16,44 @@ Add-ons
 --------
 You can install add-ons individually or all at once by running::
 
-    python -m pip install featuretools[complete]
+    python -m pip install "featuretools[complete]"
```

Comment:
The reason for the quotes is because some shells (like zsh) use square brackets for pattern matching while other shells (like bash) don't. Adding quotes makes it work for all shells.